### PR TITLE
add 70. Climbing Stairs

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 70.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    def climbStairs(self, n: int) -> int:
+        """A function to count the number of distinct way to climb a staircase to the top.
+        Args:
+            n (int): A given an integer indicating steps to reach the top.
+        Returns:
+            int: The numer of distinct way to climb to the top.
+        """
+        
+        a = 1
+        b = 1
```
Comment (kobakobashu) [inline 70.py:14]: You should name the variables so that you can guess what they mean from their names

File: 70.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        if n <= 0:
+            return n
+        
+        for i in range(n):
+            a, b =b, a + b
```
Comment (kobakobashu) [inline 70.py:19]: = b

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request merged**